Content.Server/GameObjects/Components/Kitchen/MicrowaveComponent.cs

```diff
@@ -278,17 +278,28 @@ private void wzhzhzh()
                 }
             }
 
+            var failState = MicrowaveSuccessState.RecipeFail;
+            foreach(var id in solidsDict.Keys)
```

Comment:
To be clear - this now checks that every solid in the microwave appears in *a* recipe, and not all necessarily in the same one. Is that the desired behaviour?

Reply:
fuck it i qwit

Reply:
What I meant was: this doesn't require that all the solids are part of the same recipe. If you have a recipe that needs a clown mask, and another that needs the nuke disk, then this would say it was alright to cook both the clown mask and the nuke disk at the same time, even though they're not in the same recipe together. Just checking that that was intentional.